src/components/Frame/components/Toast/Toast.tsx

```diff
@@ -64,13 +63,13 @@ export function Toast({
   const className = classNames(styles.Toast, error && styles.error);
 
   return (
-    <ThemeProvider theme={{colorScheme: 'inverse'}}>
+    <div>
```

Comment:
**todo:** Verify this update in web.

Reply:
Not a breaking change. Should we wrap this in a `dark` themeprovider?